src/Persistence.LinnApps/Repositories/PurchaseOrderRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Linn.Purchasing.Persistence.LinnApps.Repositories
+{
+    using System;
+    using System.Linq;
+    using System.Linq.Expressions;
+
+    using Linn.Common.Persistence.EntityFramework;
+    using Linn.Purchasing.Domain.LinnApps;
+
+    using Microsoft.EntityFrameworkCore;
+
+    public class PurchaseOrderRepository : EntityFrameworkRepository<PurchaseOrder, int>
+    {
+        private readonly ServiceDbContext serviceDbContext;
+
+        public PurchaseOrderRepository(ServiceDbContext serviceDbContext)
+            : base(serviceDbContext.PurchaseOrders)
+        {
+            this.serviceDbContext = serviceDbContext;
+        }
+
+        public override IQueryable<PurchaseOrder> FilterBy(Expression<Func<PurchaseOrder, bool>> expression)
+        {
+            return this.serviceDbContext.PurchaseOrders.Where(expression).Include(o => o.Details)
+                .ThenInclude(d => d.PurchaseDelivery).AsNoTracking().AsQueryable();
```

Comment:
do we need AsQueryable() here? (Not saying we don't just curious)

Reply:
Looks like we don't when I take it out, will test! Do you happen to see what I've done wrong re integration test getting no data? It feels handler-y but not sure